test/libyul/yulSyntaxTests/invalid_type.yul

```diff
@@ -2,4 +2,4 @@
     let x: invalidType
```

Comment:
what about this in general? do we plan to change the parser to not support types at all anymore?

Reply:
It might happen that we reintroduce types at some point (@ekpyron hinted at that), for which - I suppose - the syntax would stay the same. We could of course also remove them entirely from the parser and grammar instead of checking whether they are empty - but then, I guess, it would be harder to add back.